docs/scripts/generate-props-tables-data.ts

```diff
@@ -336,3 +227,13 @@ function getPropertiesFromAllTypeData(sourceTypes: TypeFileName[]) {
   });
   return targetProps;
 }
+
+function combineCategories(propertiesByCategory, toBeCombined: Category[]) {
```

Comment:
Not a new function [code](https://github.com/aws-amplify/amplify-ui/blob/1e0f38743eb9ce70259bc8ae6bf3602b073ce6fb/docs/scripts/generate-props-tables.ts#L260). Not sure why git thought it's new.